data/mods/ssb/moves.ts

```diff
@@ -756,6 +755,51 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "Ice",
 	},
 
+	// grimAuxiliatrix
+	donotsteel: {
+		accuracy: true,
+		basePower: 0,
+		category: "Special",
+		desc: "Randomly calls SSB and status moves.",
+		shortDesc: "Randomly calls SSB and status moves.",
+		name: "Do Not Steel",
+		pp: 10,
+		priority: 0,
+		flags: {},
+		onTryMovePriority: 100,
+		onTryMove(target) {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Nasty plot', source);
+		},
+		onHit(target, source, move) {
+			const moves: MoveData[] = [];
+			for (let id in BattleMovedex) {
+				const callMove = BattleMovedex[id];
+				if (callMove.category === 'Status' || !callMove.num) {
+						moves.push(callMove);
+				}
+			}
+			let randomMove;
+			if (moves.length) {
+				moves.sort((a, b) => a.num! - b.num!);
+				randomMove = this.sample(moves).name;
+			}
+			if (!randomMove) {
+				console.log(moves);
+				return false;
+			}
+
+			this.useMove(randomMove, source);
+			// TODO: need to find a way to cancel a multihit move in effect
```

Comment:
@KrisXV I require assistance with this part pls ty <3

Reply:
I would check how Substitute handles multihit moves in gen 1 since they interact similarly to that